src/rdo.rs

```diff
@@ -278,6 +280,7 @@ pub fn rdo_mode_decision(
     let mv = match luma_mode {
       PredictionMode::NEWMV => motion_estimation(fi, fs, bsize, bo, ref_frame),
       PredictionMode::NEARESTMV => if mv_stack.len() > 0 { mv_stack[0].this_mv } else { MotionVector { row: 0, col: 0 } },
+      PredictionMode::NEARMV => if mv_stack.len() > 1 { mv_stack[1].this_mv } else { MotionVector { row: 0, col: 0 } },
```

Comment:
So, if NEARMV prediction mode is chosen, the 2nd nearest is used w/o additional info coded (that I vaguely imagined, sth like index in mv_stack or mv lists), correct?

Reply:
More recent commits in this PR have 3 NEARMV modes, where NEARMV mode is signalled and followed by an index into mv_stack.